tests/test_Gene.py

```diff
@@ -0,0 +1,83 @@
+"""Pytest for Gene methods in gene_features module."""
+
+from vhip.mlmodel.gene_features import Gene
+
+
+def test_Gene_attributes():
+    """Test code to create Gene object and initialize class attributes."""
+    test_gene = Gene("ATGCCGATT")
+    assert test_gene.seq == "ATGCCGATT"
+    assert test_gene.codon_length == 3
+
```

Comment:
Add a test for the edge case where gene length is not divisible by the codon_length

Reply:
Got it. Added check for if an Excpetion gets raised when the gene length is not divisible by codon length.